examples/rustapi_module/tests/test_datetime.py

```diff
@@ -271,3 +271,11 @@ def test_tz_class():
     assert dt.tzname() == "+01:00"
     assert dt.utcoffset() == pdt.timedelta(hours=1)
     assert dt.dst() is None
+
+@pytest.mark.skip(reason='to debug with the latest master')
```

Comment:
You should use `pytest.mark.xfail` here, not `pytest.mark.skip`

Reply:
Thanks.
I tried xfail now but `ERROR: InvocationError for command` happens. It looks pytest can't handle segfault well.

Reply:
Ah yes, I forgot this causes a segfault. Presumably this is why I didn't add the `xfail` myself in my original PR. Thanks for looking into this!